spec/ig-mcode-config.json

```diff
@@ -14,16 +14,34 @@
         "historyLink": "http://hl7.org/fhir/us/mcode/history.html",
 		"primarySelectionStrategy":
 		{
-            "strategy": "namespace",
-			"primary": ["mcode"],
+            "strategy": "hybrid",
+            "primary": ["mcode",
+                "shr.entity.Patient",
+                "shr.entity.Person",
+                "shr.entity.Practitioner",
+                "shr.medication.MedicationStatement",
+                "shr.medication.MedicationAdministered",
+                "shr.procedure.LaboratoryProcedurePerformed",
+                "shr.procedure.SurgicalProcedurePerformed",
+                "shr.procedure.DiagnosticImagingPerformed",
+                "shr.onclogy.HER2ReceptorStatus"],
```

Comment:
there's a typo here. Should be shr.oncology. On that note, why are we including HER2ReceptorStatus? Testing purposes only?

Reply:
Actually, there is no more shr.oncology. I changed it to brca because the namespace is really breast cancer specific.

Reply:
Hmm... Seems like the filter should throw some errors when we reference non-existing namespaces or elements.

Reply:
testing purposes.